datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/VersionController.java

```diff
@@ -38,9 +45,14 @@
      * lists latest versions of files in user's private space.
      */
     @GetMapping(value = "/versioned/{path:.*}", produces = APPLICATION_JSON_VALUE)
+    @ApiOperation("List latest documents in user's private space")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "List command successfully completed"),
+            @ApiResponse(code = 403, message = "Access denied")
```

Comment:
403 has default message, not necessary to add it

Reply:
removed